presidio-analyzer/presidio_analyzer/predefined_recognizers/__init__.py

```diff
@@ -1,50 +1,70 @@
 """Predefined recognizers package. Holds all the default recognizers."""
 
-from presidio_analyzer.predefined_recognizers.transformers_recognizer import (
+# NER recognizers
+from presidio_analyzer.predefined_recognizers.ner.transformers_recognizer import (
     TransformersRecognizer,
 )
+from .ner.azure_ai_language import AzureAILanguageRecognizer
```

Comment:
azure ai language should be a generic, not NER, as it's functioning differently from the others which are the exit point for the NlpEngine